fbw-a380x/src/systems/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -957,6 +957,59 @@ class V1Offtape extends DisplayComponent<{ bus: EventBus }> {
   }
 }
 
+class ArsBar extends DisplayComponent<{ bus: ArincEventBus }> {
+  private arsPath = Subject.create<string>('');
+
+  private arsVisiblity = Subject.create<string>('hidden');
+
+  private airSpeed = new Arinc429Word(0);
+
+  private conf1FSelected = false;
+
+  private readonly ARS1FSpeed = 212;
+
+  private readonly Conf1FVFE = 222;
```

Comment:
These should probably be `private static readonly ARS_1_F_SPEED`. If they're instance members they should be camel case rather than pascal case.